scss/navbar.scss

```diff
@@ -0,0 +1,27 @@
+@import 'colors';
+$nav-height: 90px;
+$nav-logo-height: 30px;
+$text-big: 24px;
```

Comment:
I think this is repeated in multiple views, you should add a `sizes.scss` file with the font size to avoid repeating code.